birdhouse/components/proxy/conf.d/cors.include.template

```diff
@@ -0,0 +1,42 @@
+    proxy_hide_header 'Access-Control-Allow-Origin';
+
+    set $access_control_allow_origin "${BIRDHOUSE_PROXY_CORS_ALLOW_ORIGIN}";
+    if ( $access_control_allow_origin ~ "^$" ) {
+       set $access_control_allow_origin '*';
+    }
```

Comment:
So this fix is required because the if-statement did not work?  

If the if-statement had worked, it would already set `$access_control_allow_origin` to `*` if that value is empty initially.  Of the uninitialised warning is also generated by the if-statement itself?